About.html

```diff
@@ -0,0 +1,113 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha384-eTrSftZ6n9d5hvWbI5bvm6Hs2g2CQy2bX7X9+2zZ53D4LZ1a+Cy2OoEZ0XyNzlfj" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <title>2023 AGRICUTURAL SUMMIT</title>
+</head>
+<body>
+    <header>
+        <ul class="head-links">
+            <li>
+              <a href="#"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+                <i class="fab fa-twitter"></i>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#" class="login-btn">Logout</a></li>
+          </ul>
+        </header>
+
+        <nav>
+            <div class="ham"><i class="fa fa-bars"></i></div>
+           <div class="buger">
+            <p class="closeBtn"><i class="fa fa-times"></i></p>
+            <ul class="nav-js">
+                <li class="nav-jsi"><a href="index.html">Home</a></li>
+                <li class="nav-jsi"><a href="about.html">About</a></li>
+                <li class="nav-jsi">Program</li>
+                <li class="nav-jsi">Join</li>
+                <li class="nav-jsi">Sponsors</li>
+                <li class="nav-jsi news">News</li>
+            </ul>
+        </div>
+            <img src="images/LOGO.jpeg" alt="" class="main-logo">
+          <ul class="nav-ul">
+            <li class="nav-li"><a href="index.html">Home</a></li>
+            <li class="nav-li">Program</li>
+            <li class="nav-li">Join</li>
+            <li class="nav-li">Sponsors</li>
+            <li class="nav-li news">News</li>
+          </ul>
+        </nav>
+        <section class="aboutpg">
+        <div>
+        <h3 class="gtg">"Hello World! No Agric, No life"</h3>
+        <h1 class="topic">2023 AGRICUTURAL SUMMIT.<br>THEME:<br>SUSTAINNABLE LIVING:<br>PRACTICAL SOLUTIONS FOR<br> A GREENER FUTURE</h1>
+        </div>
+        <div class="details">
+        <p class="pg">The Global Agricultural Summit serves as a crucial platform that unites a diverse community of experts, academics, and activists who form the creative commons affiliate network. With the aim of addressing pressing agricultural challenges, this influential event takes place in a different country every two years. In each edition, the summit brings together thought leaders and stakeholders from various backgrounds, fostering collaboration and knowledge exchange.</p>
+        <p class="contactus">Please contact us per Email for any futher questions</p>
+            <p>About AGRICULTURAL SUMMIT 2023</p>
+            <a href="" class="d-link">agrisummit@gmail.com</a>
+    </div>
+    </section>
```

Comment:
- [ ] `Required`
- [ ] As mentioned earlier try to make the highlighted areas as close as to the original template
- [ ] please see screenshots below

`your template`
![image](https://github.com/Danny-imo/Capstone-project/assets/66230306/0525384f-c5f9-4397-b665-506a430f1172)

`original template`
![image](https://github.com/Danny-imo/Capstone-project/assets/66230306/e49356f5-bf54-4b1a-980c-74f39bf8be0e)